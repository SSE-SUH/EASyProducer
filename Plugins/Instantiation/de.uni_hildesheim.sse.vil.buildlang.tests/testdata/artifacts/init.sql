CREATE DATABASE test;
CREATE TABLE $test::pString;
CREATE TABLE ${test::pString + "MyTable"};
${IF test::pBoolean}INSERT INTO TABLE $test::pString${ELSE}DELETE FROM TABLE $test::pString${ENDIF};
SELECT FROM TABLE $pString WHERE "column=\$1";