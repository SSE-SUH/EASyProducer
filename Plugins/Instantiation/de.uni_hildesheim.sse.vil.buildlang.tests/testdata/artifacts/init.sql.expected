CREATE DATABASE test;
CREATE TABLE Hello;
CREATE TABLE HelloMyTable;
INSERT INTO TABLE Hello;
SELECT FROM TABLE Hello WHERE "column=$1";